### PR TITLE
Updating recent companies/projects (March 2023)

### PR DESCRIPTION
**Pull request by beta-ziliani**

### CODE REVIEW

File: _data/companies.csv [hunk 1]
```diff
@@ -14,14 +16,20 @@ NamingMatters,http://www.namingmatters.com/,SaaS products,"Trademark pre-clearan
 PlaceOS,https://place.technology/,SaaS products,"Connects everything in your office, tenancy or building to create a seamless user experience"
 Rainforest QA,https://www.rainforestqa.com/,SaaS products,"QA at the Speed of Development, Intelligent Crowdtesting for Agile Teams"
 Void,https://voidapp.co/,SaaS products,"Read + Retain in a distracted world"
+Aveine,https://www.aveine.com/,Software Development,"The first smart wine aerator"
 Bulutfon,https://www.bulutfon.com/,Software Development,"Cloud Voice & VOIP solution for Turkey"
 Duo Design,https://www.duodesign.co.uk/,Software Development,"Web and application design and development from Manchester"
+Kemal,https://kemalcr.com/,Software Development,"Fast, Effective, Simple Web Framework"
+Kinko booking,https://kinkobooking.com,Software Development,"Gérez toutes vos Réservations en un Seul Click"
```
Comment (straight-shoota) [inline _data/companies.csv:23]: The description is not much use for non-French speakers and it's a non-descriptive marketing slogan.
There's a much more meaningful definition on their homepage:
> Kinko est votre nouvelle plateforme de réservation en Real Time entre Hôtels / Maisons d'hôtes et Agences de voyages. 
Which I'd loosely translate as this:

```suggestion
Kinko booking,https://kinkobooking.com,Software Development,"A platform for managing real-time reservations in hotels and travel agencies."
```

File: _data/companies.csv [hunk 2]
```diff
@@ -14,14 +16,20 @@ NamingMatters,http://www.namingmatters.com/,SaaS products,"Trademark pre-clearan
...
 Lechner Knowledge Center,https://lechnerkozpont.hu/en/oldal/cultural-heritage-protection,Software Development,"Cultural heritage inventory"
 Level UP Solutions,https://www.levups.com,Software Development,"Web application design and development, in France, specialized in e-commerce"
 LI-COR Biosciences,https://www.licor.com/,Software Development,"LI-COR Biosciences designs and manufactures instrument systems and research tools for environmental and biotechnology research"
+Lunar,https://lunar.fyi/,Software Development,"The defacto app for controlling monitors"
```
Comment (straight-shoota) [inline _data/companies.csv:27]: The main product seems to be actually implemented in Swift. According to their submission, they're using Crystal for an adapter to control virtual displays over a network connection.
Not sure what's the best way to represent that here, but we should not insinuate the entire app is written in Crystal.

File: _data/companies.csv [hunk 3]
```diff
@@ -39,3 +47,4 @@ Concentric,https://concentric.health,Misc,"Transforming how we make decisions ab
 Kadalu Technologies,https://github.com/kadalu/moana,Misc,"An opinionated distributed Storage solution based on [GlusterFS](https://github.com/gluster/glusterfs)"
 muse-dl,https://github.com/captn3m0/muse-dl,Misc,"An ebook downloader for Project MUSE, which stitches them together into a single-file using `pdftk`"
 Nginx Shiny Exporter,https://github.com/serge-name/nginx-prometheus-shiny-exporter,Misc,"An exporter for [Prometheus](https://prometheus.io/) monitoring"
+Phenopolis,https://www.phenopolis.com/,Misc,"Advancing science and medicine through quality software."
```
Comment (straight-shoota) [inline _data/companies.csv:50]:
```suggestion
Phenopolis,https://www.phenopolis.com/,Misc,"Image processing and Big Data analysis for medical science"
```

**Review by straight-shoota — COMMENTED** [review]
I don't like to just copy marketing catchphrases with low information content. It's much more useful to clearly describe the basic idea of the products/projects. Like the first sentence in an encyclopedia article.

File: _data/companies.csv (hunk 2 above)
Comment (straight-shoota) [reply _data/companies.csv:27]: I don't think the Crystal component is really a display driver, nor is it for macOS.
It seems to be a DDC-over-HTTP relay for Raspberry Pi?

**Comment by beta-ziliani** [comment]
I agree. Yet it's worth mentioning that several existing descriptions are not specific to the Crystal part (e.g., we build ... tracks)

File: _data/companies.csv (hunk 2 above)
Comment (straight-shoota) [reply _data/companies.csv:27]: I think the main problem I have with this is that is suggests the entire app is written in Crystal, when it's actually just an adapter component.

Comment (beta-ziliani) [reply _data/companies.csv:27]: "but the UI" 🤣 

Now seriously, what can be an intermediate way to put it?

Comment (straight-shoota) [reply _data/companies.csv:26]: Maybe?
```suggestion
Lunar,https://lunar.fyi/,Software Development,"Client adapter for monitor control over HTTP"
```

**Review by straight-shoota — APPROVED** [review]

**Pull request merged**